app/scripts/modules/core/loadBalancer/filter/loadBalancer.filter.component.ts

```diff
@@ -47,16 +56,19 @@ class LoadBalancerFilterCtrl {
     app.loadBalancers.onRefresh($scope, () => this.initialize());
 
     $scope.$on('$destroy', $rootScope.$on('$locationChangeSuccess', () => {
-      LoadBalancerFilterModel.activate();
+      filterModel.activate();
       loadBalancerFilterService.updateLoadBalancerGroups(app);
     }));
+
+    $scope.$on('$destroy', () => this.groupsUpdatedSubscription.unsubscribe());
```

Comment:
Since when converting to a TS component we just have one `$onDestroy()` function and in React we have one `componentWillUnmount()`, let's just have one `$on('$destroy')`